src/app/SharpRaven/Data/SentryRequestFactory.cs

```diff
@@ -35,7 +35,9 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
-
+#if net35
+using System.Web;
```

Comment:
Good point. The reason for not using `System.Web` is that we don't want that to be a hard requirement for the library. It should only pick up the HTTP context if it's available, which is it's done in the convoluted way it is. There's a lot of situations where you don't want or can't have a reference to `System.Web`, so that's why we should avoid it. Having it in .NET 3.5 might be a fine compromise, though.